go/chunks/chunk_store.go

```diff
@@ -67,6 +67,10 @@ type ChunkStore interface {
 type Factory interface {
 	CreateStore(ns string) ChunkStore
 
+	// CreateStoreFromCache allows caller to signal to the factory that it's
+	// willing to tolerate an out-of-date ChunkStore.
+	CreateStoreFromCache(ns string) ChunkStore
```

Comment:
Yeah. This isn't particularly elegant, but I figure we aren't yet firm on exactly what contract we want here yet.